app/controllers/cycles/plugin_relations_controller.rb

```diff
@@ -40,6 +40,12 @@ def past_versions
     detail_repository.past_versions_desc(@petition.id)
   end
 
+  helper_method :color_as_rgba
+  def color_as_rgba(color, alpha)
```

Comment:
I think this method does not belong to the controller either. What if I'm writing an email body and need to convert a hex color to rgba?

I think this belongs to a view helper (in Rails) but what I mean is any other module helper. Which I can use wherever I want.